tests/test_base_scanner.py

```diff
@@ -219,7 +219,9 @@ def test_excluded_paths_is_empty_if_not_specified(self):
     def test_exclude_paths_are_calculated_if_specified(
         self, mock_compile: mock.MagicMock
     ):
-        self.options.exclude_path_patterns = ("foo",)
+        self.options.exclude_path_patterns = tuple(
+            [{"path-pattern": "foo", "reason": "Testing exclude path pattern"}]
+        )
```

Comment:
Ditto above. The only reason I comment is that it isn't obvious to the casual reader that you're getting a tuple consisting of a single dict vs a tuple consisting of a single list